testing/validator-mock/validator_client_mock.go

```diff
@@ -46,108 +45,108 @@ func (m *MockValidatorClient) EXPECT() *MockValidatorClientMockRecorder {
 	return ret0, ret1
 }
 
 // BeaconBlock indicates an expected call of BeaconBlock.
-func (mr *MockValidatorClientMockRecorder) BeaconBlock(ctx, in any) *gomock.Call {
+func (mr *MockValidatorClientMockRecorder) BeaconBlock(arg0, arg1 any) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "BeaconBlock", reflect.TypeOf((*MockValidatorClient)(nil).BeaconBlock), ctx, in)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "BeaconBlock", reflect.TypeOf((*MockValidatorClient)(nil).BeaconBlock), arg0, arg1)
 }
 
 // CheckDoppelGanger mocks base method.
-func (m *MockValidatorClient) CheckDoppelGanger(ctx context.Context, in *eth.DoppelGangerRequest) (*eth.DoppelGangerResponse, error) {
+func (m *MockValidatorClient) CheckDoppelGanger(arg0 context.Context, arg1 *eth.DoppelGangerRequest) (*eth.DoppelGangerResponse, error) {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "CheckDoppelGanger", ctx, in)
+	ret := m.ctrl.Call(m, "CheckDoppelGanger", arg0, arg1)
 	ret0, _ := ret[0].(*eth.DoppelGangerResponse)
 	ret1, _ := ret[1].(error)
 	return ret0, ret1
 }
 
 // CheckDoppelGanger indicates an expected call of CheckDoppelGanger.
-func (mr *MockValidatorClientMockRecorder) CheckDoppelGanger(ctx, in any) *gomock.Call {
+func (mr *MockValidatorClientMockRecorder) CheckDoppelGanger(arg0, arg1 any) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "CheckDoppelGanger", reflect.TypeOf((*MockValidatorClient)(nil).CheckDoppelGanger), ctx, in)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "CheckDoppelGanger", reflect.TypeOf((*MockValidatorClient)(nil).CheckDoppelGanger), arg0, arg1)
 }
 
 // DomainData mocks base method.
-func (m *MockValidatorClient) DomainData(ctx context.Context, in *eth.DomainRequest) (*eth.DomainResponse, error) {
+func (m *MockValidatorClient) DomainData(arg0 context.Context, arg1 *eth.DomainRequest) (*eth.DomainResponse, error) {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "DomainData", ctx, in)
+	ret := m.ctrl.Call(m, "DomainData", arg0, arg1)
 	ret0, _ := ret[0].(*eth.DomainResponse)
 	ret1, _ := ret[1].(error)
 	return ret0, ret1
 }
 
 // DomainData indicates an expected call of DomainData.
-func (mr *MockValidatorClientMockRecorder) DomainData(ctx, in any) *gomock.Call {
+func (mr *MockValidatorClientMockRecorder) DomainData(arg0, arg1 any) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "DomainData", reflect.TypeOf((*MockValidatorClient)(nil).DomainData), ctx, in)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "DomainData", reflect.TypeOf((*MockValidatorClient)(nil).DomainData), arg0, arg1)
 }
 
 // Duties mocks base method.
-func (m *MockValidatorClient) Duties(ctx context.Context, in *eth.DutiesRequest) (*eth.ValidatorDutiesContainer, error) {
+func (m *MockValidatorClient) Duties(arg0 context.Context, arg1 *eth.DutiesRequest) (*eth.ValidatorDutiesContainer, error) {
```

Comment:
Are all these `arg0`,`arg1`code generated changes from mockgen or something?

Reply:
Yes it should be all from mockgen run